backend/debug/dwarf/dwarf_ocaml/dwarf_type.ml

```diff
@@ -1628,12 +1648,13 @@ let rec flatten_shape (type_shape : Shape.t) (type_layout : Layout.t) =
        Otherwise, we will create an additional DWARF entry for it. *)
     | Base b -> [Known (type_shape, b)])
   | Record { fields = [(_, _, ly)]; kind = Record_unboxed }, _ ->
-    (* CR sspies: silenced error, should be handled properly instead:
-
-       Misc.fatal_errorf "unboxed record at different layout than its field,
-       record layout: %a, \ field_layout: %a" Layout.format type_layout
-       Layout.format ly *)
-    unknown_base_layouts type_layout
+    if !Clflags.dwarf_pedantic
+    then
+      Misc.fatal_errorf
+        "unboxed record at different layout than its field,\n\
+        \       record layout: %a,  field_layout: %a" Layout.format type_layout
```

Comment:
Weird line break, I would join the string back together and have ocamlformat reformat it.